### PR TITLE
USWDS 2.9.0

### PR DESCRIPTION
**Pull request by thisisdano**
# What’s new in USWDS 2.9.0

## New components
We have three new components in this release. Learn more about each of them on our website:

- **[Identifier](https://designsystem.digital.gov/components/identifier):** The identifier communicates a site's parent agency and displays agency links required by federal laws and policies.
- **[Step indicator](https://designsystem.digital.gov/components/step-indicator):** A step indicator updates users on their progress through a multi-step process.
- **[Time picker](https://designsystem.digital.gov/components/form-controls/#time-picker):** A time picker helps users select a specific time.

## Improvements and bug fixes
**Added flex-align-self utilities.** Now you can apply the `align-self` property to an element using flex-align-self utilities. The align-self rule is conceptually similar to flex-align, except `align-self` applies at the flex child level instead of the flex parent level. Thanks @taylorsolomon! (https://github.com/uswds/uswds/pull/3588)

- **flex-align-self-start:** align element at the start of the parent's cross axis
- **flex-align-self-middle:** align element in the middle of the parent's cross axis
- **flex-align-self-end:** align element at the end of the parent's cross axis
- **flex-align-self-stretch:** stretch across the full extent of the parent's cross axis

**Added an accent-warm button variant.** Now you can use `accent-warm` buttons with `usa-button--accent-warm`. (https://github.com/uswds/uswds/pull/3623)

**Generating SHA-256 hashes for our ZIP releases.** We're now generating a SHA-256 hash with the ZIP download and documenting this hash in our repo and on our website. Anyone can now check the integrity of a release ZIP file by generating a hash for their local file and comparing it to the hash we post in the github repo and on our site. (https://github.com/uswds/uswds/pull/3577)

**Proper disabled radio buttons.** We fixed the display of disabled radio buttons so both the label and radio button both appear disabled. Thanks @brayfe! (https://github.com/uswds/uswds/pull/3613)

**Proper secondary button active state.** We fixed the active state of the secondary color button. Thanks @maya! (https://github.com/uswds/uswds/pull/3615)

**Improved styling of lists inside alerts.** We improved the styling of simple and complex lists inside alerts. (https://github.com/uswds/uswds/pull/3507)

**More consistent styling of legend element.** We improved the styling of the  usa-legend element to better match other labels in a form, and improved the spacing between legends and both radio buttons and checkboxes. We added a `usa-label--large` variant to mimic the older, less consistent styling for backward compatibility. (https://github.com/uswds/uswds/pull/3465)

## Dependencies and security

package | old | new
-- | -- | --
@babel/preset-env | 7.11.0 | 7.11.5
@types/node | 14.0.27 | 14.11.1
axe-core | 4.0.1 | 4.0.2
eslint | 7.6.0 | 7.9.0
mocha | 8.1.1 | 8.1.3
prettier | 2.1.1 | 2.1.2
sass | 1.26.10 | 1.26.11
stylelint | 13.6.1 | 13.7.1
typescript | 3.9.7 | 4.0.3
yargs | 15.4.1 | 16.0.3

`0` vulnerabilities in regular dependencies (dependencies for USWDS projects installed with `npm install uswds`)
`3 low, 2 high` vulnerabilities in devDependencies (development dependencies)

**Release ZIP SHA-256 hash:** `4b3928e5a292ee4a2ac0b1a5106c179c4cfadd9355e56f5fb6b8e6e1954cfdb2`

### CODE REVIEW

File: src/stylesheets/theme/styles.scss [hunk 1]
```diff
@@ -51,8 +52,8 @@
 // @import 'packages/usa-nav-container';
 // @import 'packages/usa-nav';
 // @import 'packages/usa-navbar';
-// @import 'packages/usa-section';
 // @import 'packages/usa-search';
+// @import 'packages/usa-section';
 // @import 'packages/usa-sidenav';
```
Comment (mejiaj) [inline src/stylesheets/theme/styles.scss:57]: I think we're missing `packages/_usa-step-indicator.scss`

**Review by mejiaj — CHANGES_REQUESTED** [review]
Pushed a change to remove comment. We're missing the step-indicator package. I think after these changes we'll have to regenerate hash.

File: src/stylesheets/theme/styles.scss (hunk 1 above)
Comment (thisisdano) [reply src/stylesheets/theme/styles.scss:57]: I will be pleased to be done with this file...

Comment (thisisdano) [reply src/stylesheets/theme/styles.scss:57]: may as well add identifier as well

Comment (mejiaj) [reply src/stylesheets/theme/styles.scss:57]: Yeah, I wish we could glob it and bring only the components in.

**Review by mejiaj — APPROVED** [review]

**Pull request merged**